### DISCUSSION TITLE
About 3D Information for PCQM4M-LSC

### DISCUSSION THREAD

**Discussion by Ranceeeee** [snap-stanford/ogb#373] [PCQM4M-LSC]
Hello OGB team, I saw your 3D information about all the molecules in the training set provided by the competition. For test-challenge dataset, can we get 3D information and use it ourselves?

**Comment by weihua916** [comment]
Hi! This is a great question. You can as long as you follow the [rule](https://ogb.stanford.edu/docs/lsc/rules/).
Your test inference time, including the time to generate 3D structure, should not exceed the budget.

**Reply by PierreHao** [reply]
hi @weihua916 , that means we could use rdkit to get the 3D position if time allows? But I have found the information "test-time inference needs to be performed without explicit 3D information".

**Reply by weihua916** [reply]
Yes, you can use rdkit or whatever package, as long as it is within the budget.

**Reply by weihua916** [reply]
> But I have found the information "test-time inference needs to be performed without explicit 3D information".

This means your test input is 2D molecular graph (SMILES string) and no 3D information is given, but your model can use 3D information if you can generate it yourself.
